Controller/Adminhtml/Product/Export.php

```diff
@@ -44,6 +52,13 @@ public function __construct(
     public function execute(): void
     {
         $websiteId = $this->_request->getParam('website_id');
+        if (empty($websiteId)) {
+            $websites = $this->storeManager->getWebsites();
+            foreach ($websites as $website) {
+                $this->product->exportProducts($website->getId());
```

Comment:
what are the "regions" of each website in this case? what is the website id?

Reply:
We have been using website id up to this point. So if there is a website_id param the logic will be the same. If the button is clicked in the "default scope", there will be no website_id param so we will loop through all websites and use the existing logic. There is a check in the logic to check that the website config is Enabled for each website. 
Magento has stores within websites which share the website catalog but allow localization. The store level data is used for "regions" in current logic which will remain unchanged. This is why there is no API key config in the "store scope" as we are using the "website scope" config keys.